common/test/src/View/MapDocumentTest.cpp

```diff
@@ -1255,53 +1626,123 @@ namespace TrenchBroom {
             PreferenceManager::instance().resetToDefault(Preferences::TextureLock);
         }
 
-        // https://github.com/TrenchBroom/TrenchBroom/issues/3117
```

Comment:
I prefer if you don't delete these; I use them to get more information on why we added a particular test.

In this case you could move it to `THEN("The containining entity node is visible") {` which is the part relevant to https://github.com/TrenchBroom/TrenchBroom/issues/3117

Reply:
Agreed, I have moved it.